pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -823,4 +945,16 @@ def options(self, extra_options):
                 "particle shape '{}' not recognised".format(options["particle shape"])
             )
 
+        if options["particle cracking"] not in [
+            None,
+            "no cracking",
+            "cathode",
+            "anode",
+            "both",
+        ]:
+            raise pybamm.OptionError(
```

Comment:
To cover this, add a line either in `test_base_battery_model.py` (see `test_options` in it) and add a test that catches the error when you pass a `bad cracking model`.

Reply:
Strange because I have added one like in lines 169-171 of `test_base_battery_model.py`
```
        # crack model
        with self.assertRaisesRegex(pybamm.OptionError, "particle cracking"):
            pybamm.BaseBatteryModel({"particle cracking": "bad particle cracking"})
```

Reply:
Oh, I see. Then I think you need to add a line in `test_options` in `test_lithium_ion_parameters.py` and that should fix it. I think with this, the testing of the parameters and Rob's fix for the DFN & other tests the coverage should be pretty much sorted out.